1-exercises/D-do-while/exercise.js

```diff
@@ -7,9 +7,24 @@
 */
 
 function evenNumbersSum(n) {
-    // TODO
+    let i = 0;
+    let sum = 0;
+    // for (let i = 0; i < n; i++) {
+    //     sum += i;
+    //     console.log(`this's number is: ${n}`);
+    // }
+    do {
+        sum += i;
+        i++;
+        console.log(`this's number is: ${n}`);
+    } while (i <= n);
+    return sum;
 }
 
 console.log(evenNumbersSum(3)); // should output 6
 console.log(evenNumbersSum(0)); // should output 0
-console.log(evenNumbersSum(10)); // should output 90
\ No newline at end of file
+// console.log(evenNumbersSum());
+console.log(evenNumbersSum(10)); // should output 90
```

Comment:
You need to re-read the question. You need the loop to add up the first n even numbers, starting at 0. So if n=3, you need to add up 0+2+4. If n=10 you need to add up 0+2+4+6+8+10+12+14+16+18.